workshops/rock_paper_scissors/README.md

```diff
@@ -0,0 +1,274 @@
+        print("Tie!")
+        print('Computer =',PC)
+        print(name,'=',You)
+    elif player == "r":
+        if computer == "p":
+            print("You lose!","Paper covers Rock")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!","Rock smashes Scissors")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "p":
+        if computer == "s":
+            print("You lose!", "Scissors cut Paper")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!","Paper covers Rock")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "s":
+        if computer == "r":
+            print("You lose...","Rock smashes Scissors")
+            PC+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+        else:
+            print("You win!", "Scissors cuts Paper")
+            You+=1
+            print('Computer =',PC)
+            print(name,'=',You)
+    elif player == "reset":
+      You=1*0
+      PC=1*0
+      print("The score has been reset!")
+    else:
+        print("That's not a valid play. Please select a valid option!")
+    player = False
+    computer = t[randint(0,2)]
+
+```
+
+### 6) The End
+**And congratulations! You just made your own game using Python!**
+
+![Image](https://cloud-b32jvmwl1.vercel.app/congo.gif)
+
+If you haven't created an account on [repl.it](https://repl.it), make sure you do so to save this wonderful piece of creation!
+
+If you are facing difficulties signing up watch [this](https://www.youtube.com/watch?v=Mtqp4CUepk0).
+
+Here are some things which you can do:
+
+- Consider changing and adding more features!
+- Making it a two player game.
+- Make the game only of 3 turns instead of running it infinitely.
+- You can give the text a typewriter effect. (If you dont know how to do it watch [this][typewritter] video)
+
+**Examples on how people have customized it:**
+
+- [Fayd](https://repl.it/@FaisalSayed1/Rock-Paper-Python#main.py)
+- [CrazyBlackStone](https://repl.it/repls/ThornyWoozyBlogs#main.py)
+- [Vedant](https://repl.it/@Vedantutekar812/BEASTEXAUDOS#main.py)
+
+[typewritter]: https://youtu.be/2h8e0tXHfk0
+
+Now that you have finished building your game, you should share your beautiful creation with other people! Remember, it's as easy as giving them your URL!
+
+You probably know the best ways to get in touch with your friends and family, but if you want to share your project with the worldwide Hack Club community there is no better place to do that than on Slack.
+
+1. In a new tab, open and follow [these directions][slack] to signup for our Slack.
+2. Then, post the link to the [`#scrapbook`](https://hackclub.slack.com/messages/scrapbook) channel to share it with everyone!
+
+[slack]: https://slack.hackclub.com/
+
```

Comment:
are these extra spaces below this line? u may remove them if they aren't required! :smile: